libminifi/src/sitetosite/RawSocketProtocol.cpp

```diff
@@ -299,23 +299,32 @@ bool RawSiteToSiteClient::handShake() {
     return false;
   }
 
+  std::string error;
+
   switch (code) {
     case PROPERTIES_OK:
       logger_->log_debug("Site2Site HandShake Completed");
       peer_state_ = HANDSHAKED;
       return true;
     case PORT_NOT_IN_VALID_STATE:
+      error = "in invalid state";
+      break;
     case UNKNOWN_PORT:
+      error = "unknown";
```

Comment:
can we change this to "an unknown port" ?

Reply:
Fixed